polycommit/src/marlin_pc/mod.rs

```diff
@@ -204,7 +202,7 @@ impl<E: PairingEngine> PolynomialCommitment<E::Fr> for MarlinKZG10<E> {
         let powers = pp.powers_of_g[..=supported_degree].to_vec();
         // We want to support making up to supported_degree queries to committed
         // polynomials.
-        let powers_of_gamma_g = pp.powers_of_gamma_g[..=(supported_degree + 1)].to_vec();
+        let powers_of_gamma_g = pp.powers_of_gamma_g[..=supported_hiding_bound].to_vec();
```

Comment:
To align with the other comment: 
```suggestion
        let powers_of_gamma_g = pp.powers_of_gamma_g[..=(supported_hiding_bound + 1)].to_vec();
```